CI workflow not properly running unit tests

It appears the `./build/test.sh` file is not actually finding and running the unit tests, so it always shows a pass:

```
Running unit tests:
?   	github.com/honeycombio/honeycomb-kubernetes-agent	[no test files]

Checking go vet:
PASS
```

To reproduce:

- intentionally create a failing test, e.g. in `metrics_test.go` for `TestCpuMetrics` change `assert.Equal(t, 0.015338365, metrics[MeasureCpuUsage].GetValue(), "CPU Usage")` to `assert.Equal(t, 365, metrics[MeasureCpuUsage].GetValue(), "CPU Usage")`
- CI shows pass (snippet above from [circle](https://app.circleci.com/pipelines/github/honeycombio/honeycomb-kubernetes-agent/441/workflows/8f6128ff-babf-4cb6-aba8-73e3b29c7d38/jobs/470>))

- `go test` from metrics subdirectory shows fail

```
--- FAIL: TestCpuMetrics (0.00s)
    metrics_test.go:175: 
                Error Trace:    metrics_test.go:175
                Error:          Not equal: 
                                expected: int(365)
                                actual  : float64(0.015338365)
                Test:           TestCpuMetrics
                Messages:       CPU Usage
```

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -18,7 +18,7 @@ jobs:
       - run: go install github.com/google/ko@latest
       - run: echo "VERSION=$(cat version.txt | tr -d '\n')" >> $BASH_ENV
       - run: ./build/build.sh
-      - run: ./build/test.sh
+      - run: ./build/test.sh .
       - run: e2e-tests/test.sh
       - run:
           name: "Deploy on tagged master push"

